TitanRobotics2024/src/main/java/frc/robot/Subsystem/DriverController.java

```diff
@@ -5,32 +5,37 @@
 import frc.robot.Data.ButtonMap;
 import frc.robot.Data.PortMap;
 
-public class DriverController implements Subsystem 
+public class DriverController extends XboxController implements Subsystem
 {
     private XboxController xboxController;
     private static DriverController instance = null;
+    
+    boolean bReleased;
```

Comment:
Make the variable name more describe than bReleased, that makes no sense

Reply:
yes it does